src/content/examples/en/Parellel_Loading_Promise_all/code.js

```diff
@@ -0,0 +1,46 @@
+// PromiseAllExample.js
+
+// Declare variables to hold the images we'll load
+let img1, img2, img3;
+
+async function setup() {
+  // Add screen reader-friendly text description
+  textOutput();
+  
+  // Create a canvas where the images will be drawn
+  createCanvas(600, 400);
+
+  // Set background color to gray
+  background(220);
+
+  // Configure text appearance
+  textAlign(CENTER, CENTER);
+  textSize(18);
+
+  // Use async/await with Promise.all to load all three images at once
+  // This waits untill ALL images are loaded before continuing
+  [img1, img2, img3] = await Promise.all([
+    loadImageAsync('https://picsum.photos/100/100?random=1'), // Replace the image links with user wanted images.
```

Comment:
Thanks for the detailed documentation here! Could you explain the logic a bit more behind the helper `loadImageAsync` please? I tried the example and `loadImage` here works just as well, and the helper function does not actually have any other params - if you had another idea with this, please let me know, but if it's ok with you, I'd suggest the shorter example with only `loadImage` here.